package.json

```diff
@@ -23,6 +23,7 @@
     "jotai": "^2.10.3",
     "maplibre-gl": "^4.7.1",
     "msw": "^2.7.0",
+    "next-themes": "^0.4.4",
```

Comment:
This was added via the `npx` script, when you added the colour-mode components, right?

Reply:
Yes - not really sure why this is needed but evidently it's part of the package - I simply followed the docs here: https://www.chakra-ui.com/docs/styling/dark-mode